sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -78,6 +78,11 @@ class BigQueryEngineAdapter(InsertOverwriteWithMergeMixin, ClusteredByMixin, Row
                 exp.DataType.build("DATETIME", dialect=DIALECT),
             },
         },
+        coerceable_types={
```

Comment:
Should we call this `coerceable_type_overrides`?

Reply:
Trying to better communicate there is already an established way to define coerceable types

Reply:
i think it's fine as is

Reply:
This doesn't override though. `support_coercing_compatible_types` still applies

Reply:
Ah right thanks it just updates the set if overlapping keys. Thanks.